Effects/Deband.hlsl

```diff
@@ -0,0 +1,104 @@
+// Deband
+// Port from https://github.com/haasn/gentoo-conf/blob/xor/home/nand/.mpv/shaders/deband.glsl
+
+//!MAGPIE EFFECT
+//!VERSION 2
+
+
+//!PARAMETER
+//!DEFAULT 64
+//!MIN 0
+// The threshold of difference below which a pixel is considered to be part of
+// a gradient. Higher = more debanding, but setting it too high diminishes image
+// details.
+float threshold;
+
+//!PARAMETER
+//!DEFAULT 8
+//!MIN 0
+// The range (in source pixels) at which to sample for neighbours. Higher values
+// will find more gradients, but lower values will deband more aggressively.
+float range;
+
+//!PARAMETER
+//!DEFAULT 4
+//!MIN 0
+// The number of debanding iterations to perform. Each iteration samples from
+// random positions, so increasing the number of iterations is likely to
+// increase the debanding quality. Conversely, it slows the shader down.
+// (Each iteration will use a multiple of the configured range, and a
+// successively lower THRESHOLD - so setting it much higher has little effect)
+float iterations;
+
+//!PARAMETER
+//!DEFAULT 48
+//!MIN 0
+// (Optional) Add some extra noise to the image. This significantly helps cover
+// up remaining banding and blocking artifacts, at comparatively little visual
+// quality. Higher = more grain. Setting it to 0 disables the effect.
+float grain;
+
+//!TEXTURE
+Texture2D INPUT;
+
+//!SAMPLER
+//!FILTER POINT
```

Comment:
POINT or LINEAR? As far as I know, mpv shaders only supports linear sampling.

Reply:
Yeah you're right. It should be linear.